### PR TITLE
Use information from sysinfo.gap to locate gap and gac

### PR DESCRIPTION
**Pull request by fingolfin**
... at least when it is available (requires GAP >= 4.12). If not, fall back to what was done before, and assume they are in a specific place in the GAPROOT.

The advantage of the new approach is that we basically only need to point the package build system at sysinfo.gap, and it will derive everything else it needs from that. This then makes it easy for e.g. Linux package distributions to package GAP in a way that is compatible with their various specifications and requirements as to where to place files.

The same could be done for Browse, but I am not aware of a git repository for that? (CC @ThomasBreuer)

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -21,12 +21,19 @@ if [ -e "$GAPPATH/sysinfo.in" ]; then
   exit
 fi
 
+# set default values for GAP and GAC (may be overwritten by sysinfo.gap)
+GAP='$(GAPPATH)/bin/gap.sh'
+GAC='$(GAPPATH)/gac'
+
```
Comment (frankluebeck) [inline configure:27]: Why is this needed at all? Can it happen that $GAP or $GAC are not set in `sysinfo.gap`?

Also: Wouldn't `GAP='$(GAPPATH)/gap` be a better default?

Comment (fingolfin) [reply configure:27]: `GAP` and `GAC` are only set in GAP 4.12 and newer, so the defaults are needed for 4.9-4.11.

To me `GAP='$(GAPPATH)/gap` and `GAP='$(GAPPATH)/bin/gap.sh` are equally fine, I simply picked the one which was closed to the existing code, to minimize the differences.

Comment (frankluebeck) [reply configure:27]: Thanks for the info.
